jobs/JREGIONAL_NEXUS_GFS_SFC

```diff
@@ -18,6 +18,9 @@
 . $USHdir/source_util_funcs.sh
 source_config_for_task "cpl_aqm_parm|task_nexus_gfs_sfc" ${GLOBAL_VAR_DEFNS_FP}
 . $USHdir/job_preamble.sh "TRUE"
+
+# export COMINgfs=${COMINgfs:-$(compath.py gfs/${gfs_ver})}
+
```

Comment:
Remove this line if it is not necessary.

Reply:
We will do a global development code clean up in a later commit. Because the AQM package in its current state, still under development.

Reply:
It doesn't make sense to me. Do you mean that this PR is not complete? You should not request a review for a PR under development. If this is under development, please change the status of this PR to 'Draft'.